modules/cluster-limitations-local-zone.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-aws-localzone.adoc
+
+[id="cluster-limitations-local-zone_{context}"]
+= Cluster limitations in AWS Local Zones
+
+Some limitations exist when you attempt to deploy a cluster with a default installation configuration in Amazon Web Services (AWS) Local Zones. If you do not not configure your cluster for deployment in AWS Local Zones, edge worker nodes might not process user workloads. 
+
+The following list details limitations when deploying a cluster in AWS Local Zones:
+
+- A higher cluster-wide network maximum transmission unit (MTU) value than the MTU specified for edge worker nodes can impact how edge worker nodes in your cluster communicate. Settings a lower cluster-wide network MTU can resolve this issue. 
+- The Network Load Balancer (NLB), the Classic Load Balancer, and Nat Gateways can impact the deployment of your cluster in an AWS Local Zone. AWS Outposts do not support these networking components. Deploying a cluster in an AWS Local Zones environment requires the Application Load Balancer.
+- The `gp3-csi` storage class is not globally available in Local Zones. AWS Outposts do not support Amazon Block Store (EBS) `gp3` volumes. AWS Outposts support the `gp2-csi` storage class.
```

Comment:
```suggestion
- The AWS Elastic Block Storage (EBS) `gp3`, the default for node volumes and Storage Class on AWS deployments, is not globally available on Local Zones locations. The nodes running in Local Zones are deployed with `gp2` EBS volume by default, and the StorageClass `gp2-csi` must be set when creating workloads on Local Zone nodes.
```

Reply:
Updated to :+1: 
> The `gp3` AWS Elastic Block Storage (EBS), which is the default storage class for node volumes and AWS deployments, is not globally available on Local Zone locations. By default, the nodes running in Local Zones are deployed with the `gp2` EBS volume. The `gp2-csi` storage class must be set when creating workloads on Local Zone nodes.

Reply:
>  which is the default storage class for node volumes and AWS deployments, 

This statement is not clear/correct. The storage class is a cluster component, and the node volumes are not related to Storage Class, it shares the same technology (EBS Volume) but for Kube world is not related. Having said that, the order I suggested makes difference:
- "the default for node volumes 
- and Storage Class on AWS deployments,"

Reply:
Thanks, updated.